pkg/virt-api/webhooks/validating-webhook/admitters/migration-create-admitter_test.go

```diff
@@ -289,3 +230,21 @@ var _ = Describe("Validating MigrationCreate Admitter", func() {
 		)
 	})
 })
+
+func newAdmissionReviewForVMIMCreate(migration *v1.VirtualMachineInstanceMigration) (*admissionv1.AdmissionReview, error) {
+	migrationBytes, err := json.Marshal(&migration)
+	if err != nil {
+		return nil, err
+	}
+
+	ar := &admissionv1.AdmissionReview{
+		Request: &admissionv1.AdmissionRequest{
+			Resource: webhooks.MigrationGroupVersionResource,
+			Object: runtime.RawExtension{
+				Raw: migrationBytes,
+			},
+		},
+	}
+
+	return ar, err
```

Comment:
nit: Since this is the happy path:
```suggestion
	return ar, nil
```

Reply:
Done.